src/common/TrackedOp.h

```diff
@@ -121,6 +121,7 @@ class OpTracker {
    * @return True if there are any Ops to warn on, false otherwise.
    */
   bool check_ops_in_flight(std::vector<string> &warning_strings);
+  bool check_ops_in_flight(std::vector<string> &warning_strings, int &slow);
```

Comment:
@jcsp @gregsfortytwo 

The initial commit I made was using pointers as your guys suggested. But Yan also suggested another way. You can see my changes and comments to change this to reference above. So which one do you think is better? Change it back to pointer?

Thanks.